test/watcher.js

```diff
@@ -858,17 +859,46 @@ group('chokidar is installed', function (beforeEach, test, group) {
 			});
 		});
 
-		test('ignores dependencies outside of the current working directory', function (t) {
+		test('allows default exclusion patterns to be overriden', function (t) {
```

Comment:
Once we land the API refactor, I would like to move all the logic of building include/exclude patterns from the config into `lib/ava-files`, and rewrite many of these tests so they are just:

```js
t.true(avaFiles.shouldWatch('some/path'));
```